listenbrainz/webserver/static/js/src/playlists/Playlist.tsx

```diff
@@ -111,8 +111,9 @@ export default class PlaylistPage extends React.Component<
     });
   }
 
-  async componentDidMount() {
-    // this.connectWebsockets();
+  componentDidMount(): void {
+    this.connectWebsockets();
```

Comment:
Let's be super clean and add a `componentWillUnmount` method and call `if(this.socket?.connected){this.socket.disconnect()}` in it to make sure we terminate the connection and free up the resources.